src/sql/query.rs

```diff
@@ -1836,6 +1836,62 @@ fn plan_function<'a>(
                 Ok(expr)
             }
 
+            "round" => {
+                if sql_func.args.is_empty() || sql_func.args.len() > 2 {
+                    bail!(
+                        "round expects 1 or 2 arguments, got {}",
+                        sql_func.args.len()
+                    );
+                }
+
+                let expr1 = plan_expr(ecx, &sql_func.args[0], None)?;
+
+                if sql_func.args.len() == 1 {
+                    // When there is only one arguments, the first number has to be decimal or unknown.
+                    let floatdec_expr = match promote_number_floatdec(ecx, "round", expr1.clone()) {
+                        Ok(expr) => expr,
+                        _ => expr1,
+                    };
+                    let typ1 = ecx.column_type(&floatdec_expr);
+
+                    let expr = match typ1.scalar_type {
+                        ScalarType::Float32 => floatdec_expr.call_unary(UnaryFunc::RoundFloat32),
+                        ScalarType::Float64 => floatdec_expr.call_unary(UnaryFunc::RoundFloat64),
+                        ScalarType::Decimal(_, s) => {
+                            floatdec_expr.call_unary(UnaryFunc::RoundDecimal(s))
+                        }
+                        _ => bail!("round({}) does no exist", typ1.scalar_type),
```

Comment:
Nit: s/no/not/g

Reply:
Add one more test `SELECT round(true)` to cover the this error message.